src/components/Calendar.js

```diff
@@ -87,29 +121,51 @@ class Calendar extends React.Component {
     return weeks;
   };
 
-  _renderMonthTable = (styles) => {
+  _renderMonthTable = styles => {
     const weeks = this._getWeeks();
 
     return weeks.map(week => {
       return (
         <div key={week} style={styles.calendarWeek}>
           {week.map(day => {
-            const isCurrentMonth = day.isSame(moment.unix(this.state.currentDate), 'month');
-            const isSelectedDay = day.isSame(moment.unix(this.props.selectedDate), 'day');
+            const isCurrentMonth = day.isSame(
+              moment.unix(this.state.currentDate),
+              'month'
+            );
+            const isSelectedDay = day.isSame(
+              moment.unix(this.props.selectedDate),
+              'day'
+            );
             const isToday = day.isSame(moment(), 'day');
-            const disabledDay = this.props.minimumDate ? day.isBefore(moment.unix(this.props.minimumDate), 'day') : null;
+            const disabledDay = this.props.minimumDate ?
+              day.isBefore(moment.unix(this.props.minimumDate), 'day') :
+              null;
+            const savedStartDate = day.date();
+
 
             return (
               <div
+                className='calendar-day'
+                id={day.isSame(moment.unix(this.state.focusedDay), 'day') ? 'focused-day' : null}
                 key={day}
-                onClick={disabledDay ? null : this._handleDateSelect.bind(null, day.unix())}
-                style={Object.assign({},
+                onClick={() => {
+                  if (!disabledDay) this.props.onDateSelect(day.unix());
```

Comment:
Can disabledDay be removed as an outside dependency for this function?

Reply:
so something like this, perhaps: 
```
                onClick={
                  disabledDay ?
                    () => {} :
                    () => this.props.onDateSelect(day.unix())
                }
```

Reply:
It's probably ok as is, just trying to find ways t make these functions more self contained.